lib/matplotlib/cm.py

```diff
@@ -65,12 +66,54 @@ def _gen_cmap_d():
     # Generate reversed cmaps.
     for cmap in list(cmap_d.values()):
         rmap = cmap.reversed()
+        cmap._global = True
+        rmap._global = True
         cmap_d[rmap.name] = rmap
     return cmap_d
 
 
-cmap_d = _gen_cmap_d()
-locals().update(cmap_d)
+class _DeprecatedCmapDict(MutableMapping):
```

Comment:
Very minor naming quibble, can we put the word "Wrapper" in this class?  It took me a bit of thought to understand that this is holding a reference to `_cmap_registry` rather than having its own state.

Feel free to disregard this comment if you don't think it makes things clearer.

Reply:
No problem. This shouldn't really be in the code base for very long anyways.